docs/src/main/tut/json.md

```diff
@@ -31,9 +33,9 @@ community.  The functionality is similar:
 
 ```scala
 libraryDependencies += Seq(
-  "org.http4s" %% "http4s-argonaut" % "{{< version >}}",
+  "org.http4s" %% "http4s-argonaut" % http4sVersion,
   // Optional for auto-derivation of JSON codecs
-  "com.github.alexarchambault" %% "argonaut-shapeless_6.2" % "1.2.0-M4"
+  "com.github.alexarchambault" %% "argonaut-shapeless_6.2" % "{{< version "argonaut-shapeless_6.2" >}}"
```

Comment:
It's relevant in 0.17 because http4s-argonaut is relevant on 0.17.  It's just a pointer to the library that lets you do codec deriviation in Argonaut.

Whether it belongs in the tut is a different question.  Unless I've forgotten something, this is just an offhand remark about how to do it, and not a dependency that actually gets used in the tuts anywhere.  And whatever effort we're putting into improving JSON handling documentation, I'd probably rather first see go into circe.